### DISCUSSION TITLE
Issue but im on mobile and filling out the form is hard. Mesh values showing two different values depending where you look

### DISCUSSION THREAD

**Discussion by Tapap84** [Jyers/Marlin#1056] [General Discussion]
Manual mesh 3x3 4.2.2

set mesh of center point which would be location 1 1. I set the point to -0.02. When I go to mesh viewer it shows as -0.03. If I go to edit mesh it shows at -0.02. Only does this for center point.

Also z offset or current z height is showing wrong. I'm printing first layer. It's 0.20mm. With previous version it would show I'm on layer 0.20. With z offset at 0.00 it's showing first layer at 0.19. If I set z offset to -0.01 then its showing first layer at 0.18.

None of these seem major but it's something different and I just wanted to let you know

**Comment by gothcha** [comment]
This is not an issue, just a rounding of the values to two decimal places. It always shows the Z height relative to the mesh, so first layer will display 0.2. The corrections are applied  behind  the scenes.
